net/znc/Config.in

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (C) 2010-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+menu "Configuration"
+  depends on PACKAGE_znc
+
+config ZNC_ICU
+  bool
+  prompt "Enable ICU character encoding support."
+  help
+    Allows configuration of the charset used between clients and ZNC, and
+    between ZNC and the IRC server".
+
+endmenu
```

Comment:
This file is currently nonfunctional, you need to include it from the Makefile. Add something like
```
define Package/znc/config
        source "$(SOURCE)/Config.in"
endef
```